arbnode/schema.go

```diff
@@ -13,10 +13,14 @@ var (
 	sequencerBatchMetaPrefix     []byte = []byte("s") // maps a batch sequence number to BatchMetadata
 	delayedSequencedPrefix       []byte = []byte("a") // maps a delayed message count to the first sequencer batch sequence number with this delayed count
 
-	messageCountKey        []byte = []byte("_messageCount")        // contains the current message count
-	delayedMessageCountKey []byte = []byte("_delayedMessageCount") // contains the current delayed message count
-	sequencerBatchCountKey []byte = []byte("_sequencerBatchCount") // contains the current sequencer message count
-	dbSchemaVersion        []byte = []byte("_schemaVersion")       // contains a uint64 representing the database schema version
+	messageCountKey                 []byte = []byte("_messageCount")                       // contains the current message count
+	lastPrunedMessageResultKey      []byte = []byte("_lastPrunedMessageResultKey")         // contains the last pruned message result key
+	lastPrunedBlockHashInputFeedKey []byte = []byte("_lastPrunedBlockHashInputFeedPrefix") // contains the last pruned block hash input feed key
+	lastPrunedMessageKey            []byte = []byte("_lastPrunedMessageKey")               // contains the last pruned message key
+	lastPrunedRlpDelayedMessageKey  []byte = []byte("_lastPrunedRlpDelayedMessageKey")     // contains the last pruned RLP delayed message key
```

Comment:
I think there is an elegant way to only have one lastPrunedMEssage Key and one lastPrunedRlpDelayedMessage Key (and only update that key off all pruning ended succesfully). Not terribly important if it's too hard to do.